Automattic/WP-Job-Manager
# Fetch remote promoted template

Fixes https://github.com/Automattic/WP-Job-Manager/issues/2464

### Changes proposed in this Pull Request

* Fetch template from wpjobmanager.com
* Store in cache for 1 day
* Add function to check if template is stored
* Hide `Promote` column if there is no template to show (if our site is down, then none of these actions will work)

### Testing instructions

* You need to change the URL from `wpjobmanager.com` to whatever your local test site is
* What I did was install `wpjmcom-plugins` on my test site.  Make sure to have PHP >= 8.1 or else your site will explode
* `wpjmcom-plugins` needs to be using the `feature/promoted-jobs` branch which has the right API
* Go to the Jobs Listings page, make sure the modal works as expected

## Notes

Right now I'm hiding the entire promote column if the promote template is not available.  My thinking is that if the site is not accessible, likely none of the actions will work, is that right?
